taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -0,0 +1,231 @@
+import { Typography, Snackbar, Alert, Box, TextField, Grid, Stack, Button, IconButton, Paper, FormControl, InputLabel } from "@mui/material";
+import useFetch from "../../components/useFetch";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import { API_URL } from "../../constants";
+import Container from '@mui/material/Container';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import EditIcon from '@mui/icons-material/Edit';
+import AddBoxIcon from '@mui/icons-material/AddBox';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import Select from '@mui/material/Select';
+import ISO6391 from 'iso-639-1';
+
+const SearchResults = ({query}) => {
+    const url = API_URL+`search?query=${query}`
+    const { data: nodes, isPending, isError, isSuccess, errorMessage } = useFetch(url);
+
+    const [nodeType, setNodeType] = useState('entry'); // Used for storing node type
+    const [newLanguageCode, setNewLanguageCode] = useState(null); // Used for storing new Language Code
+    const [newNode, setnewNode] = useState(null); // Used for storing canonical tag
+    const [isValidLanguageCode, setIsValidLanguageCode] = useState(false); // Used for validating a new LC
+    const [openAddDialog, setOpenAddDialog] = useState(false);
+    const [openSuccessSnackbar, setOpenSuccessSnackbar] = useState(false);
+    const navigate = useNavigate();
+
+    // Handler function for button clicks
+    const handleClick = (event, id) => {
```

Comment:
if possible replace with `Link` , and lets avoid generic naming such as `handleClick`

Reply:
I removed the handleClick function, in favour of `Link`.